tests/eas/generic.py

```diff
@@ -322,7 +319,7 @@ def plot_expected_util(self, experiment, util_df):
                 prev = time
 
         figname = os.path.join(experiment.out_dir, 'expected_placement.png')
-        pl.savefig(figname, bbox_inches='tight')
+        fig.savefig(figname, bbox_inches='tight')
```

Comment:
This should better be on a different patch... either by itself, or eventually squased into "tests/eas/generic: Close exp_placement figure" which seems another fix... in this last case please update the commit message too.

Reply:
AFAIR this change allows us to drop the `pylab` import, hence why it's in this commit.

Reply:
Well, in that case, since we already use `pl.savefig` all around in the analysis modules... perhaps removing the pylab module perhaps should be a self contained patch which does the same all around. Do you think that's possible? At least we keep consistency on how we save generated plots...

Reply:
Ok, I'll keep that for another PR.